ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/vr/RDXVRWholeBodyKinematicStreaming.java

```diff
@@ -337,14 +353,59 @@ private void handleRightControllerJoystickInput(boolean rightAButtonPressed, boo
 
       if (enableDemonstrationButton.get())
       {
-         performingDemonstration.set(rightBButtonState);
+         if(rightBButtonPressed)
+         {
+            System.out.println(performingDemonstration.get() ? "performing" : " stopped");
+            performingDemonstration.set(!performingDemonstration.get());
+         }
```

Comment:
I do want to improve this a bit, the button label should reflect this & it should turn to false when not controlling robot

Reply:
done